### DISCUSSION TITLE
Delete borg.conf from the mastercontainer

### DISCUSSION THREAD

**Discussion by emelbardis** [nextcloud/all-in-one#1227] [Questions] [answered]
Changed my external backup drive and now get an error on manual backup test telling me to delete borg.conf. I've looked through every relevant container and cannot find this file. Can someone give me a clue on how to remove the old backup conf and let it start over on the new drive?

Thanks

**Comment by szaimen** [comment, marked as answer]
`/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/borg.config`

**Reply by emelbardis** [reply]
Thank you
